frontend/awx/views/jobs/jobUtils.tsx

```diff
@@ -19,7 +19,19 @@ export function getJobsAPIUrl(type: string) {
   }
 }
 
-export function isJobRunning(status: string) {
+type JobStatus =
+  | 'failed'
+  | 'new'
+  | 'pending'
+  | 'waiting'
+  | 'running'
+  | 'successful'
+  | 'error'
+  | 'canceled'
+  | undefined;
+
+export function isJobRunning(status: JobStatus) {
+  if (!status) return false;
```

Comment:
In the past, if status missing, we've cast it to `'waiting'`

see: https://github.com/ansible/awx/blob/devel/awx/ui/src/screens/Job/JobOutput/JobOutput.js#L117